### PR TITLE
Added kimiuso osts,changed few nisekoi and golden time images

### PR DESCRIPTION
**Pull request by roshniRam**
Added kimiuso osts,changed few nisekoi and golden time images

#### This PR is for kimiuso, nisekoi, golden time.

- [x] This issue is for enhancement/bug fixing.
- [x] This issue is an addition to JSON database.
	- [x] Pull Request Title is of form `Added [Anime Name]`.
- [x] This addition is already on ongaku.
	- [x] This PR is an improvement to current addition.
- [x] I have checked my addition locally and it is working and won't break existing entries.
- [x] I have gone through [Contribution Guidelines](https://github.com/Anshuman-Verma/ongaku/blob/master/CONTRIBUTING.md).
- [x] This pull request will be closed if I failed to update it even once in a week.

### CODE REVIEW

**Comment by DivyeshPuri** [comment]
@roshniRam Create an issue first

**Comment by roshniRam** [comment]
Ok

**Comment by Monal5031** [comment]
@roshniRam Its better to have one thing done in one PR, also one issue per PR.
@Anshuman-Verma Its good have this point in CONTRIBUTING.md as `one issue per PR` also only one specific task should be done in a single PR, should I send a PR?

File: js/data.json [hunk 1]
```diff
@@ -578,13 +578,13 @@ endings = [
 	{
 		"link" : "http://67.159.62.2/anime_ost/nisekoi-ed6-single-souzou-diary/ftdpkmbqil/02%20-%20Souzou%20Diary.mp3",
 		"name" : "Nisekoi ED6 Single - Souzou Diary",
-		"img" : "http://cdn-ak.b.st-hatena.com/entryimage/251672494-origin-1431692155.jpg"
```
Comment (anshumanv) [inline js/data.json]: Original looks better.

File: js/data.json [hunk 2]
```diff
@@ -578,13 +578,13 @@ endings = [
...
+		"img" : "http://i.imgur.com/wiGcohC.jpg"
 	},
 
 	{
 		"link" : "http://67.159.62.2/anime_ost/nisekoi-ed7-single-taisetsu-no-tsukurikata/xsiofhrdze/02%20-%20Taisetsu%20no%20Tsukurikata.mp3",
 		"name" : "Nisekoi ED7 Single - Taisetsu no Tsukurikata",
-		"img" : "http://img.manga-sanctuary.com/news/origin/3261.jpg"
```
Comment (anshumanv) [inline js/data.json]: Original looks better

File: js/data.json [hunk 3]
```diff
@@ -1040,6 +1040,42 @@ osts = [
...
+	{
+		"link" : "http://67.159.62.2/anime_ost/shigatsu-wa-kimi-no-uso-twinkle-little-star/euxawjmtld/02%20-%20Kreisler%20Love%27s%20Joy.mp3",
+		"name" : "Shigatsu wa Kimi no Uso Original Soundtrack - Kreisler Love's Joy",
+		"img"  : "https://images8.alphacoders.com/832/832863.png"
+	},
+
+	{
+		"link" : "http://67.159.62.2/anime_ost/shigatsu-wa-kimi-no-uso-original-soundtrack-vol-1/qudigmmayf/01%20-%20Shigatsu%20wa%20Kimi%20no%20Uso.mp3",
+		"name" : "Shigatsu wa Kimi no Uso Original Soundtrack - Shigatsu wa Kimi no Uso",
+		"img"  : "http://i1.wp.com/inews.gtimg.com/newsapp_match/0/1273726548/0",
```
Comment (anshumanv) [inline js/data.json]: extra comma

File: js/data.json [hunk 4]
```diff
@@ -1040,6 +1040,42 @@ osts = [
...
+	{
+		"link" : "http://67.159.62.2/anime_ost/shigatsu-wa-kimi-no-uso-original-soundtrack-vol-1/dosycblwjk/06%20-%20Watashi%2C%20Violinist%20na%20no.mp3",
+		"name" : "Shigatsu wa Kimi no Uso Original Soundtrack -Watashi, Violinist na no",
+		"img"  : "https://vignette2.wikia.nocookie.net/shigatsu-wa-kimi-no-uso/images/2/22/Epi14cover.jpg/revision/latest/scale-to-width-down/2000?cb=20150321015734"
+	},
+
+	{
+		"link" : "http://67.159.62.2/anime_ost/shigatsu-wa-kimi-no-uso-original-soundtrack-vol-2/cgqxlwiqag/06%20-%20Uso%20to%20Honto.mp3",
+		"name" : "Shigatsu wa Kimi no Uso Original Soundtrack -  Uso to Honto",
+		"img"  : "https://pedanticperspective.files.wordpress.com/2014/10/kaori-and-kousei.jpg"
```
Comment (anshumanv) [inline js/data.json]: Use this https://i.ytimg.com/vi/eN0M0912lrw/maxresdefault.jpg

File: js/data.json [hunk 5]
```diff
@@ -1040,6 +1040,42 @@ osts = [
...
+	},
+
+	{
+		"link" : "http://67.159.62.2/anime_ost/shigatsu-wa-kimi-no-uso-original-soundtrack-vol-2/vnwimijwem/10%20-%20Watashi%20no%20Uso.mp3",
+		"name" : "Shigatsu wa Kimi no Uso Original Soundtrack - Watashi no Uso ",
+		"img"  : "http://image.diyidan.net/post/2016/8/23/syhxW3Z9FtZZTUHQ.jpg"
```
Comment (anshumanv) [inline js/data.json]: Use this https://s26.postimg.org/x5d4e75qx/Ep19.png

**Review by anshumanv — APPROVED** [review]

**Pull request merged**